contrib/guix/guix-build

```diff
@@ -74,7 +74,7 @@ mkdir -p "$VERSION_BASE"
 ################
 
 # Default to building for all supported HOSTs (overridable by environment)
-export HOSTS="${HOSTS:-x86_64-linux-gnu arm-linux-gnueabihf aarch64-linux-gnu riscv64-linux-gnu
+export HOSTS="${HOSTS:-x86_64-linux-gnu arm-linux-gnueabihf aarch64-linux-gnu riscv64-linux-gnu powerpc64-linux-gnu powerpc64le-linux-gnu
```

Comment:
sooo maybe I'm architecture-ist, but I don't really want to support power... I don't think there are really any production level consumer devices that run on power, besides maybe really old MacBooks. And this will significantly increase space and time needed for guix builds... I'm okay with being ABLE to build guix for power, but I don't think we should ship, test, or support it. Open to being convinced...

Reply:
> Default to building for all supported HOSTs (overridable by environment)

we can simply just adjust our build scripts (`HOSTS="whatever archs we want" /contrib/guix/guix-build`) 🤷‍♂️

Reply:
I think I would prefer that guix-build only builds the HOSTS we ship by default.

Reply:
I don't mind modifying [`guix-start`](https://github.com/dashpay/dash/blob/a8e2316d6f9c6726a498bcae2c5c5d7354769511/contrib/containers/guix/scripts/guix-start) to exclude PowerPC, we can even go so far as building only on platforms we actively support (dropping 32-bit ARM and maybe, RISC-V), because it's our solution and won't conflict with upstream but I think that should be a separate PR.